### PR TITLE
Fix display issues with the artifact's timeline timestamp

### PR DESCRIPTION
**Pull request by mprahl**
Review this per commit to see the individual changes.

### CODE REVIEW

**Comment by sarah256** [comment]
The tests look like they're just failing from linting errors, if you could fix that too!

File: src/app/story/storyrow/storyrow.component.html [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <div class="node-uid-column story-row-column">
   <div class="node-display-name">{{ node.display_name | nodeDisplayName | truncate: 30 }}</div>
-  <div class="node-timestamp" *ngIf="node.timeline_timestamp; else noTimestamp">{{ node.timeline_timestamp }}</div>
+  <div class="node-timestamp" *ngIf="node.timeline_timestamp; else noTimestamp">
+    {{ node.timeline_timestamp | date: 'MMM d, y, HH:mm:ss':'+0000' }} UTC
```
Comment (sarah256) [inline src/app/story/storyrow/storyrow.component.html]: This is nit-picky for sure, but do we need a comma after the year too?

Comment (mprahl) [reply src/app/story/storyrow/storyrow.component.html]: We use commas after the year everywhere else, so I think it should be consistent. If you feel strongly about it, you can make a separate PR that fixes it everywhere.

**Comment by mprahl** [comment]
@sarah256 I fixed the linting error.

File: src/app/story/storyrow/storyrow.component.html (hunk 1 above)
Comment (sarah256) [reply src/app/story/storyrow/storyrow.component.html]: Nah I don't care that much, haha

**Pull request merged**